c_src/keyio.c

```diff
@@ -21,6 +21,8 @@ int acquire_uinput_keysink(int fd, char *name, int vendor, int product, int vers
   for (i=0; i < 256; i++) {
     ioctl(fd, UI_SET_KEYBIT, i);
   }
+  // Enable key repeat for output keyboard
+  ioctl(fd, UI_SET_EVBIT, EV_REP);
```

Comment:
I am not sure why we do not check `ioctl` return values in this `acquire_uinput_keysink` function.  Probably we should do that like what we do in other functions.

It may be out of the scope of this PR thought and we can do that in a future PR.

Reply:
Probably, though I think checking the length by the `strcpy` is also something important, since you can trivially type more than 80 characters in the `uinput-sink <name>` parameter.

Reply:
I fixed the overflow and some types in [my local branch](https://github.com/jokesper/kmonad/tree/fix-c). Though this doesn't add checks in the C code. If you want you could do it. (Either independently or as a PR against my branch)